Regression-Games/RegressionDocs
# [REG-1483] Reorganize sidebar

[See plan for new sidebar layout in Notion](https://www.notion.so/regressiongg/Docs-Site-Sidebar-Brainstorm-dfc40066a93349e99a4a2da9698ef815?pvs=4)

Changes:
* Remove "Player" section of Docs site
* Rename "Studio" section to "Developer Reference"
* Re-title sections + reorganize them per proposed notion layout
* Rename files to match their section titles
* Rename images for descriptiveness + kebab-case
* Consolidate `RGFindUtils` content into existing "Utilities For C# Bots" page which only contained `RGGizmos`
* Rename instances of "PlayTest Bot"  to "Adaptive Bot"
* Rename instances of "Validation Bot" to "Scenario Bot"
* Correct all links broken by reorg
  * Note that change-log has one or two links that no longer resolve to a valid page (removed player content). This is intentional.
* Remove `Studio FAQ` page
* Replace sleekplan feedback links with info@regression.gg and discord link
* Add best practices to README
* Add placeholder for "Importing the RG Unity Bots Package" page

Out of scope: 
* Consolidating "Creating Bots" docs into a single section whose contents change depending on a language toggle
* Changes/updates to page contents (explanations of concepts, fixing outdated information, adding APIs, etc.)

Screenshots: 
 
Light          |  Dark
:-------------------------:|:-------------------------:
![Screenshot 2023-12-05 at 8 18 10 PM](https://github.com/Regression-Games/RegressionDocs/assets/32661937/ec821174-cc01-4202-a100-5e9603209a97) |  ![Screenshot 2023-12-05 at 8 18 04 PM](https://github.com/Regression-Games/RegressionDocs/assets/32661937/18605ac8-c7b5-48e5-8b54-a565b076e75d)
